TcgPocket/Data/DesignTimeDateContextFactory.cs

```diff
@@ -8,8 +8,14 @@ public class DesignTimeDataContextFactory : IDesignTimeDbContextFactory<DataCont
 {
     public DataContext CreateDbContext(string[] args)
     {
+        string projectPath = AppDomain.CurrentDomain.BaseDirectory.Split(new String[] { @"bin\" }, StringSplitOptions.None)[0];
+        IConfigurationRoot configuration = new ConfigurationBuilder()
+            .SetBasePath(projectPath)
+            .AddJsonFile("appsettings.json")
+            .Build();
+        string connectionString = configuration.GetConnectionString(AppSettings.DefaultConnection);
         var optionsBuilder = new DbContextOptionsBuilder<DataContext>();
-        optionsBuilder.UseSqlServer(AppSettings.DefaultConnection);
+        optionsBuilder.UseSqlServer(connectionString);
```

Comment:
Try seeing if this works. If so, then please update it to use this :)

```suggestion
var configuration = new ConfigurationBuilder()
            .AddJsonFile("appsettings.json")
            .Build();
        
        var optionsBuilder = new DbContextOptionsBuilder<DataContext>();
        optionsBuilder.UseSqlServer(configuration.GetConnectionString(AppSettings.DefaultConnection));
        ```